pkg/spec3/fuzz.go

```diff
@@ -169,6 +169,21 @@ var OpenAPIV3FuzzFuncs []interface{} = []interface{}{
 		c.Fuzz(&v.ResponseProps)
 		c.Fuzz(&v.VendorExtensible)
 	},
+	func(v *Operation, c fuzz.Continue) {
```

Comment:
I need to think twice about this, but I don't have time right now.

Reply:
This specifically was copied from the v2 fuzzer https://github.com/kubernetes/kube-openapi/blob/master/pkg/validation/spec/fuzz.go#L68-L80